src/components/utility/LoadingButton/template.html

```diff
@@ -0,0 +1,4 @@
+<button class="loading-button" @click="handleClick" :disabled="loading">
+  <i class="fa fa-refresh fa-spin" aria-hidden="true" v-show="loading"></i>
+  <span class="loading-button__label"><slot>Submit</slot></span>
```

Comment:
loading-button is for "a button with loading state"... I think it's not a good name too... Do you have better advise?